src/IRCSock.cpp

```diff
@@ -1154,13 +1154,20 @@ void CIRCSock::TrySend() {
         m_iSendsAllowed--;
         bool bSkip = false;
         CString& sLine = m_vsSendQueue.front();
-        IRCSOCKMODULECALL(OnSendToIRC(sLine), &bSkip);
+
+        CMessage Message(sLine);
```

Comment:
Probably send queue should store messages instead of lines... That's tied to #1193